docs/UserGuide.md

```diff
@@ -39,6 +39,19 @@ This user guide will help you get started and understand how FABook can **seamle
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Familiarise with User Interface
+
+![Ui Breakdown](images/UIBreakDown.png)
+
+1. **Menu Bar**: Access the file, help or upcoming meetings here.
+2. **Search Bar**: Type in your commands here.
+3. **Command Display**: The execution of your commands appear here.
+4. **Result Display**: The result of your command execution appears here.
+5. **Contact Information**: The full contact information of a client appears here.
+6. **Data Storage**: This is where the data is stored and loaded from.
+
+--------------------------------------------------------------------------------------------------------------------
+
```

Comment:
I like this UI breakdown, I think 3 and 4 doesn't really fit the name but can change if agreed with in further refinements